modules/coremark/Kconfig

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig COREMARK
+	bool "Coremark benchmark"
```

Comment:
Shouldn't we style the name as "CoreMark" in the docs/helps/prompts/etc to keep it uniform?